build/pause/Makefile

```diff
@@ -12,20 +12,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.PHONY:	build push
+CFLAGS = -O2 -Wall -static
 
-TAG = 2.0
+#TRIPLE = arm-linux-musleabi
```

Comment:
Please do: 
```
ARCH?=amd64
ifeq($(ARCH),amd64)
TRIPLE?=x86_64-linux-musl
endif
ifeq($(ARCH),arm)
TRIPLE?=arm-linux-musleabi
endif
ifeq($(ARCH),arm64)
TRIPLE?=aarch64-linux-musl
endif
ifeq($(ARCH),ppc64le)
TRIPLE?=powerpc-linux-musl # Not sure about the naming here
endif
```
One shouldn't have to edit the file to cross-build, just specify args